Makefile

```diff
@@ -432,7 +432,11 @@ bundle: check-operator-version operator-sdk manifests update-skip-range kustomiz
 	$(SDK_BIN) generate kustomize manifests --apis-dir=./pkg/apis -q
 	@echo "kustomize using deployment image $(IMG)"
 	cd config/manager && $(KUSTOMIZE) edit set image $(APP_NAME)=$(IMG)
+	if [ x$(PLATFORM) = "xopenshift" ]; then \
+		sed -i 's%../default-bundle%../openshift-bundle%' config/manifests/kustomization.yaml; \
```

Comment:
because I'm old and set in my ways :-) I'm actually not sure if this is needed on modern shells. In the old days, this was considered a "portable" way of comparing strings in case the string was empty. 

But this should not happen because we set a default value for the variable, so I'll push a new version removing the x to make the code better readable.

thanks for the review.